src/EditorFeatures/CSharpTest/Completion/CompletionProviders/ObjectCreationCompletionProviderTests.cs

```diff
@@ -437,5 +438,147 @@ void M1()
 ";
             await VerifyItemExistsAsync(markup, "object");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task CommitOnAngleBracket_Simple()
+        {
+            var before = @"
+namespace NS
+{
+    public class A { }
+}
+
+public class C
+{
+    private NS.A _a = new $$
+}
+";
+            var after = @"
+namespace NS
+{
+    public class A { }
+}
+
+public class C
+{
+    private NS.A _a = new NS.A<
+}
+";
+            await VerifyProviderCommitAsync(before, "NS.A", after, '<', string.Empty);
```

Comment:
Right. I haven't made any of those changes yet. I was just looking at the options today. I'll try to move the tests over next week.